src/main/resources/config.yml

```diff
@@ -90,4 +90,16 @@ motd:
     idle: gray                                           # Color designating a match which is waiting to start
     starting: yellow                                     # Color designating a match which is starting
     running: green                                       # Color designating a match which is currently running
-    finished: red                                        # Color designating a match which has ended
\ No newline at end of file
+    finished: red                                        # Color designating a match which has ended
+
+groups:
+  admin:
+    priority: 1
+    prefix:
+      symbol: ❖
+      color: gold
+  mod:
+    priority: 2
+    prefix:
+      symbol: ❖
+      color: red
```

Comment:
Maybe this should go in a different `groups.yml` config file, but for now it's probably fine if it stays here

Reply:
Ok for now.